packages/medusa/src/migrations/1663059812400-linte-item-original-item-relation.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
question: should we combine this with the other migration for order edits in order to have just a single migration (as we've previously done for datamodel changes during feature development)

Reply:
I did went for that because some user are very good spy and can see a feature flag popping and then there migrations would have already ran. I prefer separate it to avoid such trouble just as a precaution measure. Once a migration have been pushed to develop I would go with writing a new migration for any changes to the schema